### PR TITLE
Fix dialog label valign

### PR DESCRIPTION
**Pull request by danrabbit**
## Before
Screenshot from 2020-01-22 10 47 41@2x

## After
Screenshot from 2020-01-22 10 47 07@2x

### CODE REVIEW

File: elementary/gtk-3.0/gtk-widgets.css [hunk 1]
```diff
@@ -4209,6 +4209,10 @@ dialog toolbar {
     background-color: @bg_color;
 }
 
+.dialog-action-area button > * {
+    min-height: calc(1em + 7px);
```
Comment (cassidyjames) [inline elementary/gtk-3.0/gtk-widgets.css]: Do we wanna comment how this was derived?

Comment (danrabbit) [reply elementary/gtk-3.0/gtk-widgets.css]: tbh I don't really understand why this works. I guess it's like theoretically the alignment inside the button or grid or whatever it is should be the height of the text plus padding, but then for some reason it's off by one? So maybe there's some kind of rounding? I dunno, it's magic numbers

Comment (cassidyjames) [reply elementary/gtk-3.0/gtk-widgets.css]: Good enough for me 😜

Comment (micahilbery) [reply elementary/gtk-3.0/gtk-widgets.css]: Could it be off by 1px because it's accounting for the text-shadow?

**Pull request merged**